src/card-product.ts

```diff
@@ -180,4 +182,40 @@ export class CardProductApi {
     this.client = client
   }
 
+  /*
+   * Function to take a series search arguments, most of which are optional,
+   * as input, pass them to Marqeta, and have them return any Marqeta Card
+   * Products that fit the criteria. If no search arguments are given, this
+   * returns *all* the Card Products in Marqeta.
+   */
+  async get(search: {
+    count?: number,
+    startIndex?: number,
+    sortBy?: string,
+  } = {}): Promise<{
+    success: boolean,
+    body?: CardProduct,
+    error?: MarqetaError,
+  }> {
+    const searchOptions = snakecaseKeys(search)
```

Comment:
Why create this? As you can say:

```typescript
     const resp = await this.client.fire('GET',
       'cardproducts',
       snakecaseKeys(search),
     )
```

but I'd really think about `options` on that... as it's not a search... at most, it's a "positioning"